WordPress/src/main/java/org/wordpress/android/ui/pages/PageItemViewHolder.kt

```diff
@@ -163,10 +162,6 @@ sealed class PageItemViewHolder(internal val parent: ViewGroup, @LayoutRes layou
                 radioButton.setOnClickListener {
                     onParentSelected(pageItem)
                 }
-
-                @Suppress("DEPRECATION")
-                CompoundButtonCompat.setButtonTintList(radioButton,
```

Comment:
when we remove this, the radio button has the light blue color, right? At least that's what see in the app now. Is that what we want? 😄 
<img width="475" alt="Screenshot 2019-12-11 at 14 44 00" src="https://user-images.githubusercontent.com/1079756/70626409-b0de9900-1c24-11ea-8664-0eb0444402a3.png">

Reply:
Yes, there is no reason for this radio buttons to be special ones :)